fe/src/pages/About/About.tsx

```diff
@@ -1,5 +1,3 @@
-import React from 'react';
```

Comment:
I'm not sure about removing `react` imports, altough true, that its not required for react to work.
IMHO this rises more problems with eslint than fixes, so i'd say we should add the imports back. 🤔 
![errors](https://user-images.githubusercontent.com/48998862/115368874-30448b00-a1d0-11eb-8004-e458f1b07a75.JPG)

Reply:
Actually when i restarted local project the errors were gone, so the `react/react-in-jsx-scope`  rule works after all.

I'd say then we can leave the imports as they now are.